Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -1113,7 +1113,7 @@ class AtlasStepper {
   template <typename propagator_state_t>
   Result<double> step(propagator_state_t& state) const {
     // we use h for keeping the nominclature with the original atlas code
-    auto& h = state.stepping.stepSize;
+    auto h = state.stepping.stepSize.value();
```

Comment:
not sure if the step size changes later in the code should update the state? with my current change it would not